spec/core_functions/math/acos.hrx

```diff
@@ -55,7 +55,7 @@ a {b: math.acos(1.000000000001)}
 
 <===> one_fuzzy/output.css
 a {
-  b: 0deg;
+  b: NaNdeg;
```

Comment:
shouldn't the case of being fuzzy equal to the bound be handled in `math.acos` instead of returning NaN ?

Reply:
OK, I missed https://github.com/sass/sass/commit/498fda605cb2e872312123a5ca56746d995423c5 so this is actually expected